cargo-dist/src/tasks.rs

```diff
@@ -3180,6 +3216,29 @@ fn tool_info() -> DistResult<Tools> {
         omnibor: find_tool("omnibor", "--version"),
         // Computed later if needed
         code_sign_tool: None,
+
+        // NOTE: This doesn't actually give us cargo-auditable's version info,
+        // but it does confirm it's installed, which is what we care about.
+        cargo_auditable: find_tool2("cargo", "auditable", "--version"),
+
+        cargo_cyclonedx: find_tool2("cargo", "cyclonedx", "--version"),
+        cargo_xwin: find_tool2("cargo", "xwin", "--version"),
+        cargo_zigbuild: find_tool("cargo-zigbuild", "--version"),
+    })
+}
+
+fn find_tool2(name: &str, arg: &str, test_flag: &str) -> Option<Tool> {
```

Comment:
Can we get a better description of what this does and why it's a different method from the other one?

Reply:
Renamed it to `find_cargo_subcommand`, since that's what it's for.